app/views/accounts/login.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Please sign in below</h1>
+<% if @errors %>
+<% @errors.each do |message| %>
+  <%= message %>
+<%end%>
+<%end%>
+
+<%= form_for @user, url: {action: "authenticate"}, html: {class: "nifty_form"} do |f| %>
+ <%= f.label :username %>
+  <%= f.text_field :username %>
+  <%= f.label :password %>
+  <%= f.password_field :password %>
```

Comment:
It's good practice to also add a `password_confirmation` label and field here. `has_secure_password` will deal with it if it's there.